packages/monaco/src/browser/monaco-command.ts

```diff
@@ -120,6 +126,57 @@ export class MonacoEditorCommandHandlers implements CommandContribution {
             }
         };
     }
+    protected newConfigTabSizeHandler(): MonacoEditorCommandHandler {
+        return {
+            execute: (editor: MonacoEditor) => {
+                const editorModel = editor.document;
+                if (editorModel && editorModel.textEditorModel) {
+                    const tabSize = editorModel.textEditorModel.getOptions().tabSize;
+                    const tabSizeOptions = Array.from(Array(8), (_, x) => x + 1)
+                        .map(size => new UpdateTabSizeOpenItem(size === tabSize ? `${size} *` : size.toString(), () => {
+                            editorModel.textEditorModel.updateOptions({
+                                tabSize: size,
+                                insertSpaces: editorModel.textEditorModel.getOptions().insertSpaces
+                            });
+                        })
+                        );
+                    this.open(tabSizeOptions, 'Change the Tab Size to...');
+                }
+            },
+            isEnabled: (editor: MonacoEditor) => !!editor
```

Comment:
you can remove `isEnabled`, `MonacoCommandRegistry` takes care that there is an instance of `MonacoEditor ` already